packages/keystone/databaseAdapters/BalancingReplicaKnexAdapter/adapter.js

```
@@ -13,179 +13,260 @@
 const { isRuleMatching } = require('./utils/rules')
 const { extractCRUDQueryData } = require('./utils/sql')
 
+const DEFAULT_POOL_MAX = conf['DATABASE_POOL_MAX']
+    ? parseInt(conf['DATABASE_POOL_MAX'], 10)
+    : 3;
```

ESLint

[error] Extra semicolon.

(@typescript-eslint/semi)